rfc/rfc006-distributed-registry.md

```diff
@@ -333,6 +333,25 @@ The care organisation refers to it:
 }
 ```
 
+### 4.1. Network Endpoint Service
+
+The underlying peer-to-peer network does not automatically discover new peers, but it relies on the system to tell
+to which peers to connect. Nodes SHOULD publish their Nuts Network endpoint(s) as a service in their DID Document(s).
```

Comment:
Do you mean EVERY DID Document made by a certain node should contain this endpoint? Or some? And which ones?

Reply:
Important question: does this require a relation between the certificate and the DID publishing it? If not: then can any DID doc contain such an endpoint regardless of the DID is managed by that node?